MapboxNavigationTests/BottomBannerSnapshotTests.swift

```diff
@@ -0,0 +1,59 @@
+import XCTest
+import Foundation
+import SnappyShrimp
+import MapboxDirections
+@testable import TestHelper
+@testable import MapboxNavigation
+@testable import MapboxCoreNavigation
+
+class BottomBannerSnapshotTests: SnapshotTest {
+    override func setUp() {
+        super.setUp()
+        recordMode = false
+    }
+    
+    @available(iOS 11.0, *)
+    
+    func testBottomBannerViewController() {
+        let host = UIViewController(nibName: nil, bundle: nil)
+        let container = UIView.forAutoLayout()
+        let subject = BottomBannerViewController(nibName: nil, bundle: nil)
+        
+        host.view.addSubview(container)
+        constrain(container, to: host.view)
+        
+        embed(parent: host, child: subject, in: container) { (parent, banner) -> [NSLayoutConstraint] in
+            banner.view.translatesAutoresizingMaskIntoConstraints = false
+            return banner.view.constraintsForPinning(to: container)
+        }
+        
+        applyStyling(to: subject)
+        subject.prepareForInterfaceBuilder()
+
+        verify(host, for: Device.iPhoneX.portrait)
```

Comment:
If practical, it would be very nice to also test an older, home button–laden device such as iPhone 8, to make sure there aren’t any issues like zero-height views that confuse Auto Layout.

Reply:
Also, please include the snapshots in the PR description.